express-mock-af/routes/consumption-reporting.js

```diff
@@ -0,0 +1,41 @@
+var express = require('express');
+var router = express.Router();
+
+var querystring = require('querystring');
+var querystring = require('body-parser');
+
+/* GET users listing. */
+router.post('/:aspId', function(req, res, next) {
+  const aspId = req.params.aspId;
```

Comment:
This should only return a response code, see `TS26512_M5_ConsumptionReporting.yaml`: 

````
    responses:
        '204':
          description: 'Consumption Report Accepted'
        '400':
          description: 'Bad Request'
        '415':
          description: 'Unsupported Media Type'
````

If no check is performed in the dummy server simply return 
````
 res.send(204);
````

Reply:
Note that the _aspId_ parameter is due to be replaced by _provisisioningSessionId_ in TS 26.512 V17.6.0.

See 5G-MAG/Standards#76 for links to Rel-17 **CR0037**r2. This will hopefully be approved at SA#101 Plenary next week.

Reply:
> Note that the _aspId_ parameter is due to be replaced by _provisisioningSessionId_ in TS 26.512 V17.6.0.
> 
> See [5G-MAG/Standards#76](https://github.com/5G-MAG/Standards/issues/76) for links to Rel-17 **CR0037**r2. This will hopefully be approved at SA#101 Plenary next week.

Please be advised that this M5 API change was approved at SA#101 last week.

It will be included in TS 26.512 when V17.6.0 is officially published by 3GPP.